pgc_ortho.py

```diff
@@ -87,6 +87,10 @@ def main():
     if not os.path.isdir(dstdir):
         parser.error("Error arg2 is not a valid file path: {}".format(dstdir))
 
+    # log input command for reference
+    command_str = ' '.join(sys.argv)
+    logger.info("Running command: {}".format(command_str))
```

Comment:
logger is used here before any handlers are defined. I have never tired it, but I assume this will do nothing.